FlowCal/excel_ui.py

```diff
@@ -1521,11 +1545,14 @@ def run(input_path=None,
         "--histogram-sheet",
         action="store_true",
         help="generate sheet in output Excel file specifying histogram bins")
-    args = parser.parse_args()
+    args = parser.parse_args(args=argv)
```

Comment:
Fails on Windows 10, both when using `flowcal` and `python -m FlowCal.excel_ui`. Replacing this line with the previous version, `args = parser.parse_args()`, makes everything work.

I think the reason is that the list of arguments, as given by `sys.argv`, includes the path of the script as the first element on Windows. However, `parser.parse_args` doesn't expect this. Therefore, it fails.

An immediate solution is to replace this line with the old version, and eliminate the option to specify argument `argv` to the function `run_command_line`.

Reply:
Same behavior in Linux Mint 18. It looks like the first element of `sys.argv` is **always** the script name, which `parser.parse_args` does not expect. This is true in all OS, both in [python2](https://docs.python.org/2/library/sys.html#sys.argv) and [python3](https://docs.python.org/3/library/sys.html#sys.argv).

Reply:
Indeed, it appears that `argparse` version 1.1 (the default `argparse` module included in Python 2.7 [[source](https://github.com/python/cpython/blob/2.7/Lib/argparse.py)] and Python 3.6 [[source](https://github.com/python/cpython/blob/3.6/Lib/argparse.py)]) strips out the first argument here [[source](https://github.com/python/cpython/blob/2.7/Lib/argparse.py#L1710)]:

```python
    def parse_known_args(self, args=None, namespace=None):
        if args is None:
            # args default to the system args
            args = _sys.argv[1:]
        else:
            # make sure that args are mutable
            args = list(args)
```

Thus, I think we're justified in doing the same thing. Might be nice to explicitly state all of this in a brief comment in the code, though.